editor/components/editor-history/redo.js

```diff
@@ -26,11 +18,11 @@ function EditorHistoryRedo( { hasRedo, redo } ) {
 	);
 }
 
-export default connect(
-	( state ) => ( {
-		hasRedo: hasEditorRedo( state ),
-	} ),
-	( dispatch ) => ( {
-		redo: () => dispatch( { type: 'REDO' } ),
-	} )
-)( EditorHistoryRedo );
+export default compose( [
+	withSelect( ( select ) => ( {
+		hasRedo: select( 'core/editor' ).hasEditorRedo(),
+	} ) ),
+	withDispatch( ( dispatch ) => ( {
+		redo: () => dispatch( 'core/editor' ).redo(),
```

Comment:
Minor: Can we just write: `redo: dispatch( 'core/editor' ).redo`?

Reply:
The reason I didn't do this is because we're attaching this directly to the onclick handler which means it receives the event as argument and I don't want it to be passed too the action creator. Even if in this particular case it doesn't do anything special since it just ignores all arguments, but I already had to fix a bug because of this exact same issue where we were passing an argument inadvertently.

Reply:
It makes sense, thank you for clarifying @youknowriad 👍